ecosystem/sep-0031.md

```diff
@@ -346,14 +346,14 @@ Name | Type | Description
 `amount_in` | string | (optional) The amount received or to be received by the Receiving Anchor. Up to 7 decimals are permitted. Excludes any fees charged after Receiving Anchor receives the funds.
 `amount_out` | string | (optional) The amount sent by the Rending Anchor to Receiving Client. Up to 7 decimals are permitted.
 `amount_fee` | string | (optional) The amount of fee charged by the Receiving Anchor.
-`stellar_account_id` | string | The Stellar account to send the payment to.
+`stellar_account_id` | string | The Receiving Anchor's Stellar account the Sending Anchor will be making the payment to.
 `stellar_memo_type` | string | The type of memo to attach to the Stellar payment: `text`, `hash`, or `id`.
 `stellar_memo` | string | The memo to attach to the Stellar payment.
 `started_at` | UTC ISO 8601 string | (optional) Start date and time of transaction.
 `completed_at` | UTC ISO 8601 string | (optional) Completion date and time of transaction.
-`stellar_transaction_id` | string | (optional) The transaction_id on Stellar network of the transfer that initiated the payment.
+`stellar_transaction_id` | string | (optional) The transaction_id on Stellar network of the transfer that initiated the payment from the Sending Anchor to the Receiving Anchor.
```

Comment:
I think this change is a bit redundant. The only Stellar transaction involved in a SEP-31 flow is the payment between these two parties, I don't see a reason to mention them here in addition to the `stellar_account_id` definition.

Reply:
I agree that's somewhat implicit but it wasn't very clear to me in my first read, that's why I added it. I think it's ok to leave it out of this update though.

Reply:
https://github.com/stellar/stellar-protocol/pull/962/commits/c7d4c260aa3c31b095a203a1fdc0138a5c271c53